Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -663,10 +663,16 @@ internal nfloat GetEstimatedRowHeight(UITableView table)
 				}
 
 				// We're going to base our estimate off of the first cell
-				var firstCell = templatedItems.ActivateContent(0, templatedItems.ListProxy[0]);
+				Cell firstCell;
+				var isGroupingEnabled = List.IsGroupingEnabled;
+
+				if (isGroupingEnabled)
+					firstCell = templatedItems.ActivateContent(0, templatedItems.GetGroup(0)?.ListProxy[0]);
+				else
+					firstCell = templatedItems.ActivateContent(0, templatedItems.ListProxy[0]);
 
 				// Let's skip this optimization for grouped lists. It will likely cause more trouble than it's worth.
```

Comment:
Not strictly related to this bug, but when I read this comment I expected "skipping" would imply taking the pass less taken; e.g. taking the `if` when grouping was enabled. But we actually take the if if grouping is disabled.